### ISSUE TITLE
#-1433892483------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1831]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div data-test="item-details-header" class="sc-dhKdcB jtDRtM">feed_automationTest_httpPoller_JSON_2024_02_08_0403</div>`

<strong>Element Location: </strong>div[data-test="item-details-header"]

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-1433892483_All page content should be contained by landmarks.png`
